sql/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -363,7 +363,7 @@ object SparkConnectService extends Logging {
    * Starts the GRPC Service.
    */
   private def startGRPCService(): Unit = {
-    val debugMode = SparkEnv.get.conf.getBoolean("spark.connect.grpc.debug.enabled", true)
+    val debugMode = SparkEnv.get.conf.getBoolean("spark.connect.grpc.debug.enabled", false)
```

Comment:
If Apache Spark still needs to depend on this flag somehow in testing environment, we had better make it enabled only when `SPARK_TESTING=1`.